core/build.sbt

```diff
@@ -4,18 +4,15 @@ libraryDependencies ++= Seq(
   scalaOrganization.value %  "scala-compiler" % scalaVersion.value
 )
 
-scalaVersion := "2.11.8"
+scalaVersion := "2.12.3"
 
-crossScalaVersions := Seq("2.10.6", scalaVersion.value, "2.12.2")
+crossScalaVersions := Seq("2.11.11", "2.12.3")
 
-libraryDependencies := {
-  CrossVersion.partialVersion(scalaVersion.value) match {
-    case Some((2, scalaMajor)) if scalaMajor >= 11 =>
-      libraryDependencies.value :+ "org.scala-lang.modules" %% "scala-xml" % "1.0.5"
-    case _ =>
-      libraryDependencies.value
-  }
-}
+libraryDependencies += "org.scala-lang.modules" %% "scala-xml" % "1.0.5"
+
+// scripted-plugin is enabled by default, in particular in this non-sbt subproject
+// this means that switching to 2.11.11 will result in non-existent dependencies
+libraryDependencies ~= { _.filterNot(_.organization == "org.scala-sbt") }
```

Comment:
I was speaking too early. This workaround actually breaks execution of scripted tests.